lib/appraisal/bundler_dsl.rb

```diff
@@ -114,7 +139,7 @@ def ruby_version_entry
 
       case @ruby_version
       when String then "ruby #{@ruby_version.inspect}"
-      else "ruby(#{@ruby_version.inspect})"
```

Comment:
This was a minor bug, as it was not calling `Utils.format_string`, as many of the other similar methods in this class do (and they are correct for so doing).  Some of the variability in output between Ruby versions was due to this.